src/qt/peertablemodel.cpp

```diff
@@ -162,7 +162,8 @@ QVariant PeerTableModel::data(const QModelIndex &index, int role) const
         case NetNodeId:
             return (qint64)rec->nodeStats.nodeid;
         case Address:
-            return QString::fromStdString(rec->nodeStats.addrName);
+            // prepend to peer address down-arrow symbol for inbound connection and up-arrow for outbound connection
+            return QString(rec->nodeStats.fInbound ? "↓ " : "↑ ") + QString::fromStdString(rec->nodeStats.addrName);
```

Comment:
Does those unicode char (86 93 and 86 91) work on all supported platforms (especially older windows version)?
Would using an Icon make more sense?

Reply:
Could You define 'older'?

Reply:
These characters have been in unicode since time immemorial, I wouldn't be afraid of them not being supported.